src/backend/apps/users/views.py

```diff
@@ -51,6 +52,21 @@ def post(self, request, format=None):
         return Response(status=status.HTTP_200_OK)
 
 
+class UserProfileView(views.APIView):
```

Comment:
fyi I added something like this to master, so can prob pull this -- hit `/api/auth/user/` like how it used to work